### ISSUE TITLE
Missing declaration types in 1.0.2

### ISSUE THREAD

**Issue by melchor629** [Borewit/token-types#9]
Hi, today I was building some docker images when I suddenly found the following error:

```
#10 33.94 > nas-fs@0.6.2 build /app
#10 33.94 > tsc -p .
#10 33.94 
#10 38.73 node_modules/strtok3/lib/type.d.ts(2,27): error TS7016: Could not find a declaration file for module 'token-types'. '/app/node_modules/token-types/lib/index.js' implicitly has an 'any' type.
#10 38.73   Try `npm install @types/token-types` if it exists or add a new declaration (.d.ts) file containing `declare module 'token-types';`
```

I'm using the package [music-metadata](https://github.com/borewit/music-metadata) which has a dependency that uses this package and with the 1.0.2 version I'm getting this error. On the other hand, if I force a package downgrade by issuing the command `npm install token-types@1.0.1` I can get the TS files compiled. It seems that the types are not uploaded in 1.0.2, couldn't be? Or maybe I forgot something?

Thanks.

**Comment by Borewit** [comment]
I am very sorry guys, I wasn't watching my own repo and therefor I did not notice this issue any earlier.

Thanks for letting me know, I will address this issue as soon as possible.

**Comment by Borewit** [comment]
Fixed in [v1.0.3](https://github.com/Borewit/token-types/tree/v1.0.3)
